components/match2/commons/brkts_wiki_specific_base.lua

```diff
@@ -52,7 +52,11 @@ Called from MatchGroup
 -- @returns module
 ]]
 function WikiSpecificBase.getMatchGroupContainer(matchGroupType)
-	if matchGroupType == 'matchlist' then
+	-- TODO Add a check if opponent count is > 2
+	if Lua.moduleExists('Module:GameSummary') then
```

Comment:
not sure this is the best idea
for wikis that have both normal and BR/FFA matches it will cause issues